src/documentation/pages/components/header/examples/_portal/_base_header_portal.vue

```diff
@@ -165,11 +175,11 @@ declare const chi: any;
 })
 export default class BasePortal extends Vue {
   mounted() {
-    chi.drawer(document.getElementById('drawer-trigger-pib1'));
-    chi.dropdown(document.getElementById('button-portal-site-menu'));
-    chi.dropdown(document.getElementById('button-portal-user-menu'));
-    chi.dropdown(document.getElementById('button-portal-eid-menu'));
-    chi.tooltip(document.getElementById('button-portal-notification'));
+    chi.drawer(this.$refs['drawer-trigger-portal-pib1'] as HTMLElement);
+    chi.dropdown(this.$refs['button-portal-site-menu'] as HTMLElement);
+    chi.dropdown(this.$refs['button-portal-user-menu'] as HTMLElement);
+    chi.dropdown(this.$refs['button-portal-eid-menu'] as HTMLElement);
+    chi.tooltip(this.$refs['button-portal-notification'] as HTMLElement);
```

Comment:
These chi.js component instances are not being disposed. Let's address this in another PR. Approving this one.